modules/task/backend.go

```diff
@@ -30,6 +30,12 @@ import (
 
 var gobEncoding = &GobEncoding{}
 
+const (
+	_initialized = iota
```

Comment:
hmm i think breaking the underscore rule here might be warranted. this is hard to look at